src/lumigo_tracer/parsers/utils.py

```diff
@@ -213,6 +222,18 @@ def parse_http_method(event: dict):
     return result
 
 
+def _parse_http_method_v2(event: dict):
+    result = {
+        "triggeredBy": "apigw",
+        "httpMethod": event.get("requestContext", {}).get("http", {}).get("method"),
+        "resource": event.get("requestContext", {}).get("http", {}).get("path"),
+        "messageId": event.get("requestContext", {}).get("requestId", ""),
```

Comment:
why the default here is empty string and not None like the rest? 😂

Reply:
This is how its implemented in V1, I don't know if it's for a reason